replayer/frame.cpp

```diff
@@ -187,9 +187,36 @@ void readTail(
   }
 }
 
+// The boolean array better be divisible by 8
+std::vector<uint8_t> bool_to_bytes(std::vector<bool> arr) {
+  std::vector<uint8_t> ret;
+  ret.resize(arr.size() / 8);
+  for (size_t i=0; i<arr.size(); i++) {
+    ret[i / 8] |= arr[i] << (i % 8);
+  }
+  return ret;
+}
+std::vector<bool> bytes_to_bool(std::vector<uint8_t> arr) {
+  std::vector<bool> ret;
+  ret.resize(arr.size() * 8);
+  for (size_t i=0; i<arr.size(); i++) {
+    for (size_t k=0; k<8; k++) {
+      ret[i * 8 + k] = (arr[i] >> k) & 1;
+    }
+  }
+  return ret;
+}
+
```

Comment:
`std::vector<bool>` stores its data as a bitfield already; however, it doesn't expose it this way. Maybe it's easier to just use std::vector<int8_t> everywhere that to do the bitfield conversion effectively twice?